src/include/settings/settings_callbacks.h

```diff
@@ -28,204 +28,108 @@ class Callbacks {
   static void NoOp(void *old_value, void *new_value, DBMain *db_main,
                    common::ManagedPointer<common::ActionContext> action_context);
 
-  /**
```

Comment:
I think there's limited value to copypaste comments. I left them for NoOp and removed the rest.